visual-tests/multiple-window-creation-deletion/multiple-window-creation-deletion-test.cpp

```diff
@@ -76,7 +76,7 @@ class MultipleWIndowCreationDeletionTest: public VisualTest
     // Create a text label
     Actor actor = TextLabel::New();
     actor.SetProperty(Control::Property::BACKGROUND, Color::RED);
-    actor.SetSize(100, 100);
+    actor.SetProperty( Actor::Property::SIZE, Vector2( 100, 100 ) );
```

Comment:
Change to Vector3 temporarily. At least until after your dali patches have been merged.